calico_kubernetes/calico_kubernetes.py

```diff
@@ -454,7 +475,29 @@ def _get_api_path(self, path):
 
         # The response body contains some metadata, and the pods themselves
         # under the 'items' key.
-        return json.loads(response_body)['items']
+        return json.loads(response_body)
+
+    def _patch_api(self, path, patch):
+        """
+        Patch an api resource to a given path
+
+        :param path: The relative path to an API endpoint.
+        :param patch: The updated data
+        :return: A list of JSON API objects
+        :rtype list
+        """
+        logger.info(
```

Comment:
We don't need to print out KUBE_API_ROOT all over the place - we log it out once at the beginning and it never changes.

These logs can be reduced to a single debug level log which prints out the path and the patched data.

Info level logs can be added when this method is being called to provide higher-level context.

So, it might look like this:
```
logger.info("Patching pod with Calico endpoint ID")
self._patch_api(...)
```